### PR TITLE
Support OF 1508:25

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
### Added

- Support for OF 1508:25

### Changed

- OF 1541 merged into 1508

### How to prepare for test
- [x] ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] paxa the environment: `paxa`
- [x] install on stage:
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`
- [x] install on stage:
`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-clinical-api-stage.sh [THIS-BRANCH-NAME]`

### How to test external apptags positive
- [x] do import an order with the new of 1508:25 with no-container as container type
- [x] select an external apptag for a sample 
- [x] save sample

### Expected test outcome
- [x] save ok

### How to test external apptags negative
- [x] do import an order with the new of 1508:25 with tube as container type
- [x] select an external apptag for a sample 
- [x] save sample

### Expected test outcome
- [x] save not ok
 
- [x] Take a screenshot and attach or copy/paste the output.

### test submit
- [x] 1508

### Expected outcome
- [x] fastq ok
- [x] mip-rna ok
- [x] mip-dna ok
- [x] balsamic ok

## Review
- [x] code approved by MR
- [x] tests executed by PG
- [x] "Merge and deploy" approved by PG
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [x] Document in changelog
- [ ] Deploy this branch on clinical-db
- [ ] Deploy this branch on hasta
- [ ] Deploy this cgweb branch on clinical-db
- [ ] add new OF 1508:25 in SuSy
- [ ] Inform to AG, ES

### CODE REVIEW

**Comment by patrikgrenfeldt** [comment]
Test result:
image
🆗 External application can have "No container", External application can't have "Tube"

image
🆗 Non-external application can't have "No container", Non-external application can have "Tube"

File: cg/apps/orderform/excel_orderform_parser.py [hunk 1]
```diff
@@ -22,8 +22,7 @@ class ExcelOrderformParser(OrderformParser):
     NO_VALUE: str = "no_value"
     SHEET_NAMES: List[str] = ["Orderform", "orderform", "order form"]
     VALID_ORDERFORMS: List[str] = [
-        "1508:24",  # Orderform MIP, Balsamic, sequencing only, MIP RNA
-        "1541:6",  # Orderform Externally sequenced samples
```
Comment (Mropat) [inline cg/apps/orderform/excel_orderform_parser.py:26]: this wont exist anymore?

Comment (patrikgrenfeldt) [reply cg/apps/orderform/excel_orderform_parser.py:26]: It's part of this PR to merge 1541 into 1508

**Comment by patrikgrenfeldt** [comment]
Test result fastq:
image
🆗 

Test result mip-rna:
image
🆗 

Test result mip-dna:
image
🆗 

Test result balsamic:
image
🆗 

image
🆗 subject_id saved for balsamic & mip-rna & mip-dna

**Review by Mropat — APPROVED** [review]

**Comment by patrikgrenfeldt** [comment]
Deployed on clinical-db: 
```
Installing collected packages: cg
  Attempting uninstall: cg
    Found existing installation: cg 22.24.1
    Uninstalling cg-22.24.1:
      Successfully uninstalled cg-22.24.1
Successfully installed cg-22.25.0
```

Deployed on Hasta:
```
Successfully built cg
Installing collected packages: cg
  Attempting uninstall: cg
    Found existing installation: cg 22.21.1
    Uninstalling cg-22.21.1:
      Successfully uninstalled cg-22.21.1
Successfully installed cg-22.25.0

INFO  [alembic.runtime.migration] Context impl MySQLImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
ERROR [alembic.util.messaging] Multiple head revisions are present for given argument 'head'; please specify a specific target revision, '<branchname>@head' to narrow to a specific head, or 'heads' for all heads
  FAILED: Multiple head revisions are present for given argument 'head'; please specify a specific target revision, '<branchname>@head' to narrow
  to a specific head, or 'heads' for all heads
```

**Pull request merged**